desktop/renderer-app/scripts/vite-plugin-electron/options.ts

```diff
@@ -0,0 +1,29 @@
+import { getTemplate } from "./lib";
+
+const electronCode = getTemplate("./template/electron.template.js");
+const eventsCode = getTemplate("./template/events.template.js");
+const fsExtraCode = getTemplate("./template/fs-extra.template.js");
+const extractZipCode = getTemplate("./template/extract-zip.template.js");
+
+export const options = [
```

Comment:
```suggestion
export const libraryPatchConfigs = [
```

filepath: `options.ts` => `template/index.ts`

Reply:
I suggest keep `options.ts` , we may support plugin options, and this file will be default options

Reply:
okey